src/pkcs11/framework-pkcs15.c

```diff
@@ -2741,11 +2741,17 @@ pkcs15_gen_keypair(struct sc_pkcs11_slot *slot, CK_MECHANISM_PTR pMechanism,
 		goto kpgen_done;
 
 	if (keytype == CKK_GOSTR3410)   {
+		unsigned int kp[] = {1, 2, 643, 2, 2, 35, 1, (unsigned int)-1};
+		unsigned int hp[] = {1, 2, 643, 2, 2, 30, 1, (unsigned int)-1};
```

Comment:
please use a better name for your OIDs.

Why are they needed on the PKCS#11 level?